app/src/main/java/com/epfl/drawyourpath/userProfile/cache/GoalAndProgress.kt

```diff
@@ -0,0 +1,54 @@
+package com.epfl.drawyourpath.userProfile.cache
+
+import androidx.room.ColumnInfo
+import com.epfl.drawyourpath.userProfile.UserModel
+
+data class GoalAndProgress(
+    /**

```

Comment:
I suggest to rename this class GoalAndAchievements to be convenient with the database